### ISSUE TITLE
Atom not supported in mac M1

### ISSUE THREAD

**Issue by muhammad-maged** [atom/atom#22914]

**Comment by ubunteroz** [comment]
Atom works fine on M1 (tested on Big Sur & Monterey) with Rosetta 2. It's usable.

**Comment by jwhitehorn** [comment]
I disagree. Apple Silicon Macs have been out for over a year, and by default Rosetta 2 is not installed - certainly not ideal. Additionally, in Monterey you get this dialog on first launch:

<img width="372" alt="Screen Shot 2021-11-24 at 6 28 44 PM" src="https://user-images.githubusercontent.com/523322/143329721-c175bc38-030a-446e-93a7-f5bd37243769.png">

There's obviously a few action items for the team here.
